forester/src/nullifier/nullify.rs

```diff
@@ -86,64 +92,55 @@ pub async fn nullify<T: Indexer>(indexer: T, config: &Config) -> Result<(), Fore
             .compressed_account_proofs
             .remove(&account.hash_string())
         {
-            let client =
-                RpcClient::new_with_commitment(&config.server_url, CommitmentConfig::confirmed());
             let successful_nullifications = Arc::clone(&successful_nullifications);
             let cancellation_token_clone = cancellation_token.clone();
-            let arc_indexer_clone = Arc::clone(&arc_indexer);
             let config_clone = config.clone();
-            let task = tokio::spawn(async move {
-                let _permit = permit;
-                let mut retries = 0;
-                loop {
-                    if cancellation_token_clone.is_cancelled() {
-                        info!("Task cancelled for account {}", account.hash_string());
+            let _permit = permit;
+            let mut retries = 0;
+            loop {
```

Comment:
on testnet this retry logic will not work. It will hit the rpc ratelimit immediately.
You will need a delay for retries.

Reply:
It probably makes sense to create a function send with retries.

Reply:
It has the delay here: https://github.com/Lightprotocol/light-protocol/pull/811/files/44d4ead2d6dc267dfbb4511305667fe0cd125aaa#diff-fed5e22bba6202e7a68545ef7f19570dbd3c084c5aa2f0d46703fd176a94371cR138